iocAdmin/Db/Makefile

```diff
@@ -46,9 +46,4 @@ $(COMMON_DIR)/iocAdminVxWorks.db: $(COMMON_DIR)/iocAdminScanMon.db $(COMMON_DIR)
 
 $(COMMON_DIR)/siteEnvVars.substitutions: $(EPICS_BASE)/configure/CONFIG_SITE_ENV
 	@echo Expanding siteEnvVars.substitutions from CONFIG_SITE_ENV....
-	@echo "file iocEnvVar.template" > $@
-	@echo "{" >> $@
-	@echo "pattern" >> $@
-	@echo "{ ENVNAME, ENVVAR, ENVTYPE }" >> $@
-	@sed -n 's/^EPICS_\([A-Z_]*\).*/{\1, EPICS_\1, epics}/p' $< >> $@
-	@echo "}" >> $@
+	@perl ../genSiteEnvVars.pl < $< > $@
```

Comment:
+1 on using `$(PERL)`.

Most Base build rules that run generator scripts start with `@$(RM) $@` to explicitly delete the target file first, so any errors which result in the script not actually regenerating the output file will trigger an error from GNUmake instead of it just continuing without recreating the file. I guess that's an enhancement request since the original rule didn't do that, but I recommend adding it anyway.

Switching from stdin to a filename argument might not need any changes to your Perl code at all, since using `while (<>) { ... }` in Perl inline code automatically opens and parses the files listed in `@ARGV` anyway (improve on that, Python!).